exercises/practice/spiral-matrix/spiral_matrix.h

```diff
@@ -0,0 +1,4 @@
+#ifndef SPIRAL_MATRIX_H
+#define SPIRAL_MATRIX_H
+
```

Comment:
Given the recent feedbac in other issues, should we have at least the struct definition here?

Reply:
Sure, I will add it.